src/odbc/unittests/common.h

```diff
@@ -158,8 +158,13 @@ SQLSMALLINT odbc_alloc_handle_err_type(SQLSMALLINT type);
 	CHKR2(SQLSetPos, (odbc_stmt,a,b,c), SQL_HANDLE_STMT, odbc_stmt, res)
 #define CHKSetStmtAttr(a,b,c,res) \
 	CHKR2(SQLSetStmtAttr, (odbc_stmt,a,b,c), SQL_HANDLE_STMT, odbc_stmt, res)
+#if ODBCVER >= 0x0300
+#define CHKSetStmtOption(a,b,res) \
+	CHKSetStmtAttr(a, (SQLPOINTER) b, SQL_NTS, res)
+#else
```

Comment:
I would say no, the idea is exactly to check obsolete functions and that they are correct.

Reply:
Windows builds of the `rowset` test otherwise warn
```
...\src\odbc\unittests\rowset.c(63): warning C4996: 'SQLSetStmtOption': ODBC API: SQLSetStmtOption is deprecated. Please use SQLSetStmtAttr instead.
```

Reply:
I've reworked this commit to patch `rowset.c` rather than `common.h`  and have it still call `SetStmtOption`, just with the warning suppressed.